firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsUncaughtExceptionHandler.java

```diff
@@ -46,7 +46,13 @@ public CrashlyticsUncaughtExceptionHandler(
   public void uncaughtException(Thread thread, Throwable ex) {
     isHandlingException.set(true);
     try {
-      crashListener.onUncaughtException(settingsDataProvider, thread, ex);
+      if (thread == null) {
+        Logger.getLogger().d("Could not handle uncaught exception; null thread");
```

Comment:
What do you think about making some of this output logged as errors, since they are basically logging failed asserts?

Reply:
I think in this particular case that makes sense.